SPEC.md

```diff
@@ -587,17 +587,18 @@ $import = 'import' $ws+ $string ($ws+ 'as' $ws+ $identifier)?
 
 The import statement specifies that `$string` which is to be interpted as a URI which points to a WDL file.  The engine is responsible for resolving the URI and downloading the contents.  The contents of the document in each URI must be WDL source code.
 
-If a namespace identifier (via the `as $identifer` syntax) is specified, then all the tasks and workflows imported will only be accessible through that [namespace](#namespaces).  If no namespace identifier is specified, then all tasks and workflows from the URI are imported into the current namespace.
+Every imported WDL file requires a namespace and it can be specified using an identifier (via the `as $identifer` syntax). If no namespace identifier is explicitly specified, then the default namespace is the filename of the imported WDL file. For all imported WDL files, the tasks and workflows imported from that file will only be accessible through that assigned [namespace](#namespaces)
```

Comment:
I dont explicitly mention the minus fie extension piece--but hopefully the example provided would clear that point.